resources/lang/en.json

```
@@ -8,5 +8,20 @@
     "Confirm Password": "Confirm Password",
     "Remember Me": "Remember Me",
     "Please confirm your password": "Please confirm your password",
-    "Forgot Your Password?": "Forgot Your Password?"
+    "Forgot Your Password?": "Forgot Your Password?",
+    "Name": "Name",
+    "Edit": "Edit",
+    "Delete": "Delete",
+    "Dashboard": "Dashboard",
+    "New": "New",
+    "Categories": "Categories",
+    "submit": "submit",
+    "Name is required!": "Name is required!",
+    "Successfull!": "Successfull!",
+    "Create successfull": "Create successfull",
+    "Update successfull": "Update successfull",
+    "Delete success": "Delete success",
+    "Not found category": "Not found category",
+    "Name is already": "Name is already",
```

```suggestion
    "name.unique": "Name is already existed.",
```
Tạo bản dịch dưới format ntn nhé.